spec/classes/datadog_agent_spec.rb

```diff
@@ -2319,9 +2431,17 @@
       end
       let(:facts) do
         {
-          operatingsystem: 'CentOS',
-          osfamily: 'redhat',
           facts_array: ['one', 'two'],
+          osfamily: 'redhat',
```

Comment:
```suggestion
```
You probably missed this one

Reply:
Actually not.  If I specify…
```
          facts_to_tags: ['os.family', 'facts_array'],
```
… then [`datadog_agent::tag5`](https://github.com/DataDog/puppet-datadog-agent/blob/main/manifests/tag5.pp#L8) tries to find it via…
```
     $value = getvar($tag_name)
```
… and `$os.family` does not exist.

But there could be Help:  `stdlib` got a Function called `fact()`.  So this could be changed to…
```
  if $lookup_fact{
    $value = fact($tag_name)
```
… but not in this Change.